src/fvm_multicell.hpp

```diff
@@ -194,15 +240,38 @@ class fvm_multicell {
     }
 
 private:
-
     threshold_watcher threshold_watcher_;
 
     /// current time [ms]
-    value_type t_ = value_type{0};
+    value_type t_ = 0;
 
     /// resting potential (initial voltage condition)
     value_type resting_potential_ = -65;
 
+    /// final time in integration round [ms]
+    value_type tfinal_ = 0;
+
+    /// max time step for integration [ms]
```

Comment:
`dt_max_` is closer to the meaning associated with the interface (i.e. the `cell_group` requests a maximum _dt_). A `target_dt` is closer to the meaning as provided by this lowered cell implementation, as you describe.

A comment on how `dt_max_` is satisfied is more appropriate for the integration code, and I can add one there.